python/uv/__main__.py

```diff
@@ -1,9 +1,10 @@
 import os
 import sys
-import sysconfig
 
+from uv import find_uv_bin
 
-def detect_virtualenv() -> str:
+
+def _detect_virtualenv() -> str:
```

Comment:
Marked this as private explicitly 👍 because by default in Python anything that's no prohibited (at least by convention in case of `_` is allowed).

Reply:
Do we have tests for these files? I don't see any Python test harness just yet 🤔

Reply:
Nope. We should probably `pip install uv` and `python -m uv` smoke test in CI when we build wheels for releases? We could track this separately.

Reply:
We could also probably install directly e.g. `uv pip install -e ./python/uv` and then test in the "Smoke test" step?

Reply:
Yeah, agreed, though likely should be a separate PR 😊 Up to you if you want to wait for this until that lands.

Reply:
Feel free to put that up if you're interested! :) unless we get to it first